Autopilot/Inc/TelemPathInterface.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Provides declarations common to the telemetry and path manager interfaces.
+ * Author: Anthony Bertnyk, Messed with by Gordon Fountain to turn from Attitude Manager to Telemetry
+ */
+
+#ifndef TELEM_PATH_INTERFACE_HPP
+#define TELEM_PATH_INTERFACE_HPP
+
+//Data for path manager to send to telemetry.
+typedef struct DataForTelem{
```

Comment:
The struct is named DataForTelem but in other places you use TelemData, which doesn't exist. You probably meant to name this struct TelemData.

Reply:
Changed it to be more clear about where the data comes from and is going to but forgot I had referenced it elsewhere.